app/views/languages/index.haml

```diff
@@ -27,8 +27,7 @@
                 %td= language.name_km
                 %td= timeago(language.created_at).html_safe
                 %td.text-right
-                  .d-flex.d-md-block.flex-row
-                    = render 'shared/action_buttons', edit_url: edit_language_path(language), delete_url: language_path(language), object: language, remote: true
+                  = render 'shared/action_buttons', edit_url: edit_language_path(language), delete_url: language_path(language), object: language, remote: true
```

Comment:
@hengsokly flex-row seems it used for responsive screen, please double check if removes is fine with responsive screen

Reply:
No, this class is only for saying flex-direction, but d-flex class by default it is flex-row direction